doc/aws-deployment.md

```diff
@@ -0,0 +1,386 @@
+    correct value for `MAURO_ENDPOINT`.
+
+4.  Run the `update-from-github.sh` script.  With all the above in place the
+    site should be available via the hostname
+
+5.  Work through the "Configuring Mauro" section of the Site setup
+    documentation.
+
+6.  Ingest a suitable branch and when that has completed create a new version
+
+\pagebreak
+# Configuring Mauro
+
+With the site running there are a number of tasks to carry out before it can
+be used.
+
+## Administrator account
+
+When you first access Mauro (MDM UI, not the Orchestrator), you can sign in
+with the standard Mauro account created during bootstrap:
+
+Username: "admin@maurodatamapper.com"
+Password: "password"
+
+After sign in, you will be forced to change this password to something more
+secure.
+
+## Site URL
+
+There is a configuration token used in Mauro, particularly for email templates,
+stating what the base URL is for this Mauro instance. This must be set
+manually.
+
+1.  Sign in as the administrator
+2.  Click on the user menu (top-right corner) and select "Configuration"
+3.  Click the "Add" button
+4.  Under "Select the property to add", select "site.url"
+5.  Under "Value", enter the base URL domain this Mauro instance uses.
+6.  Click "Add property" to save.
+
+## OpenID Connect
+
+The OpenID Connect feature needs to be switched on in Mauro.
+
+1.  Sign in as the administrator
+2.  Click on the user menu (top-right corner) and select "Configuration"
+3.  Click the "Add" button
+4.  Under "Select the property to add", select "feature.use_open_id_connect"
+5.  Under "Value", select "Yes".
+6.  Click "Add property" to save.
+
+## User groups
+
+1.  Sign in as the administrator
+2.  Click on the user menu (top-right corner) and select "Manage groups"
+3.  Click the "Add" button.
+4.  Create a new group called "readers", then click "Add group"
+5.  Repeat for a new group called "editors".
+
+## User accounts
+
+Once groups are added, create the user accounts.
+
+1.  Sign in as the administrator
+2.  Click on the user menu (top-right corner) and select "Manage users"
+3.  Click "Add"
+4.  Enter the details for each user (see below), including the user group they
+    belong in.
+5.  Click "Add user" to save.
+6.  Repeat for each user.
+
+**Note**: at this point in the setup, you may not have SMTP setup yet so the
+new users will not automatically get their initial password directly. It is
+possible to workaround this though:
+
+1.  Go to the "Emails" page
+2.  Each email should have a status of "Failed", but the body of each
+    registration email will be listed.
+3.  Click the "Show more" link in each row to view the initial password for
+    each user account, then send it to them manually and securely.
+4.  As each user signs in, they will be forced to change their password to
+    something of their choice.
+
+The following user accounts should be created:
+
+Name                Email                       Environments            Groups
```

Comment:
This Markdown table hasn't rendered correctly in GitHub. I think the syntax is:

```
| Name | Email | Environments | Groups |
| ------ | ------ | --------------- | -------- |
| value | value | value | value
```

Reply:
That's annoying, the simplified Markdown syntax is easier to maintain, changed